Dockerfile

```
@@ -123,7 +123,7 @@ RUN conda install -y python=3.7.1 \
                      mkl-service \
                      numpy=1.19 \
                      scipy=1.5 \
-                     scikit-learn=0.19 \
+                     'scikit-learn>=0.21' \
```

```suggestion
                     scikit-learn=0.21 \
```
We typically pin a particular minor in the Dockerfile for the trade-off between flexibility (of new patch releases fixing something) and reproducibility of the environment (expecting constant APIs through the same minor of the software)

---

Got it! Will commit this together with the `pandas` upgrade that `nilearn` apparently also requires :-)